samples/cloudwatch/README.md

```diff
@@ -76,34 +91,29 @@ cat sas-prom-cwagentconfig.yaml \
 kubectl delete pod -n amazon-cloudwatch -l app=cwagent-prometheus
 ```
 
-### Visualizing Metrics
+## Visualizing Metrics
 
-Metrics collected by the Prometheus CloudWatch agent are stored in the
+After you configure metric collection for CloudWatch, you can create dashboard widgets in CloudWatch to visualize the collected metric data. Metrics collected by the CloudWatch agent for Prometheus are stored in the
 `/aws/containerinsights/[cluster-name]/prometheus` log group.
-dashboard widgets can be created within Amazon CloudWatch.
 
-For example, to create a graph of `go_memstates_alloc_bytes`
-(memory usage) of SAS Go-based microservices:
+For example, these are the steps to create a graph of `go_memstates_alloc_bytes`
+(memory usage) of SAS Viya microservices that are written in Go:
 
-* From the AWS web console, navigate to CloudWatch
-* In the navigation bar on the left, choose 'Metrics'
-* Choose the `ContainerInsights/Prometheus` namespace
-* Choose `ClusterName, job, namespace, pod, sas_service_base`
-* Search for or find the `go_memstates_alloc_bytes` metric
-* Right-click and choose `Search for this only`
-* Select the checkbox of one or more services (rows) to add them
-to the graph at the top
+1. From the AWS web console, navigate to CloudWatch.
+2. In the navigation bar on the left, choose `Metrics`.
+3. Select the `ContainerInsights/Prometheus` namespace.
+4. Select `ClusterName, job, namespace, pod, sas_service_base`.
+5. Locate the `go_memstates_alloc_bytes` metric.
+6. Right-click on the metric and select `Search for this only`.
```

Comment:
It's a left-click on the down-arrow next to the metric name.

Reply:
Text changed